sdk/lib/atl/atlimage.h

```diff
@@ -1186,9 +1232,7 @@ class CImage
         // create a DIB section
         HDC hDC = ::CreateCompatibleDC(NULL);
         ATLASSERT(hDC);
-        LPVOID pvBits;
-        HBITMAP hbm = ::CreateDIBSection(hDC, bi.get(), DIB_RGB_COLORS,
-                                         &pvBits, NULL, 0);
+        HBITMAP hbm = ::CreateDIBSection(hDC, &bi, DIB_RGB_COLORS, NULL, NULL, 0);
```

Comment:
Are you sure that omitting pvBits pointer is valid in Windows?

Reply:
Yes, I am. No error nor problem found.